pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -38,17 +40,31 @@ const (
 var _ = ScoreWithNormalizePlugin(&TestScoreWithNormalizePlugin{})
 var _ = ScorePlugin(&TestScorePlugin{})
 
-func newScoreWithNormalizePlugin1(inj injectedResult) Plugin {
-	return &TestScoreWithNormalizePlugin{scoreWithNormalizePlugin1, inj}
+func newScoreWithNormalizePlugin1(injUnkown *runtime.Unknown, f FrameworkHandle) (Plugin, error) {
+	inj, err := decodeInjectedResult(injUnkown)
+	if err != nil {
+		return nil, err
+	}
+	return &TestScoreWithNormalizePlugin{scoreWithNormalizePlugin1, *inj}, nil
 }
-func newScoreWithNormalizePlugin2(inj injectedResult) Plugin {
-	return &TestScoreWithNormalizePlugin{scoreWithNormalizePlugin2, inj}
+func newScoreWithNormalizePlugin2(injUnkown *runtime.Unknown, f FrameworkHandle) (Plugin, error) {
```

Comment:
nit: \n

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/82483/commits/b3a82e3b93af38c478b6e65145e24e4730ef7882